Configuration/Settings.yaml

```diff
@@ -1,17 +1,21 @@
 Neos:
   Cqrs:
     EventStore:
-      storage:
-        options:
-          eventTableName: 'neos_cqrs_events'
-
-          backendOptions:
-            driver: 'pdo_mysql'
-            host: '127.0.0.1'
-            dbname: NULL
-            user: NULL
-            password: NULL
-            charset: 'utf8'
+      stores:
+        'default':
+          boundedContexts:
+            'Foo.Bar': true
+            '*': true
```

Comment:
That syntax suggests that you can do sth like `Foo.*` which I wouldn't recommend.
What if we just make the `boundedContext` optional and have the default one omit it?